src/omv/img/imlib.h

```diff
@@ -1141,6 +1141,12 @@ typedef struct find_barcodes_list_lnk_data {
     int quality;
 } find_barcodes_list_lnk_data_t;
 
+typedef enum image_hint {
```

Comment:
Can you rename these with a PREFIX. Like IMAGE_HINT_BILINEAR, IMAGE_HINT_CENTER.